docs/src/auto_examples/tutorials/run_lda.py

```diff
@@ -220,7 +219,7 @@ def extract_documents(url='https://cs.nyu.edu/~roweis/data/nips12raw_str602.tgz'
 #
 # I suggest the following way to choose iterations and passes. First, enable
 # logging (as described in many Gensim tutorials), and set ``eval_every = 1``
-# in ``LdaModel``. When training the model look for a line in the log that
+# in ``LdaModel``. Then, when training the model, look a line in the log that
```

Comment:
-1: the new version doesn't make sense.

Reply:
I agree that the for is changing the sense.

I propose:

`Then, when training the model, look for a line in the log that`

Reply:
97cce0e8